src/Haskell/Ide/Engine/Plugin/GhcMod.hs

```diff
@@ -310,9 +310,12 @@ extractRenamableTerms msg
                        . T.lines
     singleSuggestions = T.splitOn "), " -- Each suggestion is comma delimited
     isKnownSymbol t = " (imported from" `T.isInfixOf` t  || " (line " `T.isInfixOf` t
-    getEnclosed = T.dropWhile (== '‘')
-                . T.dropWhileEnd (== '’')
-                . T.dropAround (\c -> c /= '‘' && c /= '’')
+    getEnclosed' b e = T.dropWhile (== b)
+                     . T.dropWhileEnd (== e)
+                     . T.dropAround (\c -> c /= b && c /= e)
+    getEnclosed txt = case getEnclosed' '‘' '’' txt of
```

Comment:
I think a comment why we do this would be very helpful for the future!

Reply:
I've just was thinking in what message would be adequate! 😄 
But i wanted to investigate a little bit why delimiters are different in windows and linux, i is soooo strange.

Reply:
I've found https://gitlab.haskell.org//ghc/ghc/issues/2507#note_79667 (closed)
I deduce that it is related with utf support, anyway i'll simply comment that it is needed for windows.